bids/variables/variables.py

```diff
@@ -512,8 +512,17 @@ def resample(self, sampling_rate, inplace=False, kind='linear'):
         x = np.arange(n)
         num = len(self.index)
 
+        if sampling_rate < self.sampling_rate:
+            # Downsampling, so filter the signal
+            from scipy.signal import butter, filtfilt
+            b, a = butter(5, (sampling_rate / 2) / (self.sampling_rate / 2),
```

Comment:
Ah, I bet it's a integer division problem.

```suggestion
            b, a = butter(5, (sampling_rate / 2.0) / (self.sampling_rate / 2.0),
```